InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoAlign.java

```diff
@@ -25,12 +18,20 @@
     private final VisionSubsystem VISION_SUBSYSTEM;
     private final NavigationSubsystem NAVIGATION_SUBSYSTEM;
 
+    private final PIDController PID;
+
 
 
     public AutoAlign(DriveSubsystem driveSubsystem) {
         DRIVE_SUBSYSTEM = driveSubsystem;
```

Comment:
We need to add requirements to this subsystem, otherwise it'll conflict with idle.